lib/services/runtime/handlers/noMatch.ts

```diff
@@ -10,20 +10,28 @@ export type NoMatchCounterStorage = number;
 
 export const EMPTY_AUDIO_STRING = '<audio src=""/>';
 
-const removeEmptyNoMatches = (noMatchArray?: string[]) => noMatchArray?.filter((noMatch) => noMatch != null && noMatch !== EMPTY_AUDIO_STRING);
+type NoMatchNode = BaseNode.Utils.BaseNode & VoiceNode.Utils.DeprecatedNodeNoMatch & { noMatch?: Nullable<VoiceNode.Utils.NodeNoMatch> };
+
+const removeEmptyNoMatches = (node: NoMatchNode) => {
+  const noMatches = node.noMatches || node.noMatch?.noMatches || [];
+  return noMatches?.filter((noMatch) => noMatch != null && noMatch !== EMPTY_AUDIO_STRING);
+};
+
+const getElseID = (node: NoMatchNode) => node.elseId ?? node.noMatch?.nodeID ?? null;
 
 export const NoMatchHandler = () => ({
-  canHandle: (node: VoiceNode.Utils.NodeNoMatch & BaseNode.Utils.BaseNode, runtime: Runtime) => {
-    const nonEmptyNoMatches = removeEmptyNoMatches(node.noMatches);
+  handle: (node: NoMatchNode, runtime: Runtime, variables: Store) => {
+    const nonEmptyNoMatches = removeEmptyNoMatches(node);
 
-    return Array.isArray(nonEmptyNoMatches) && nonEmptyNoMatches.length > (runtime.storage.get<NoMatchCounterStorage>(S.NO_MATCHES_COUNTER) ?? 0);
```

Comment:
no more can handle. It is always able to handle if the handler that calls it gives up.